invenio_oauthclient/ext.py

```diff
@@ -127,3 +127,16 @@ def init_config(self, app):
         for k in dir(config):
             if k.startswith('OAUTHCLIENT_'):
                 app.config.setdefault(k, getattr(config, k))
+
+        # Override template configuration.
+        template_key = app.config.get(
+            'OAUTHCLIENT_TEMPLATE_KEY',
+            'SECURITY_LOGIN_USER_TEMPLATE'  # default template key
+        )
+        if template_key is not None:
```

Comment:
Looks OK upon quick look?  It would be nice to document that people can set `OAUTHCLIENT_TEMPLATE_KEY` to select their wanted template... it is not mentioned anywhere else nor in examples.  At the least describe it in the commit log, perhaps?